docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
+- Import of an existing certificate already held in z/OS to the `ZoweKeyring` for use by Zowe.  
+- Creation of a locally generated certificate and signed by an existing certificate authority. The certificate is placed in the key ring.
+
+## Where to get started with Zowe certificates?
+
+Zowe records the configuration in `zowe.yaml` file with [5 different scenarios](./certificate-configuration-scenarios.md) for you to quickly get started with.
+
+Before starting configuring your certificate, you might want to get familiar with the concepts in Zowe certificates.
+
+### Background knowledge
+
+#### Certificates in Zowe architecture
+
+Placeholder for a diagram
+
+#### Truststore
+
+Truststores are the repositories that contain cryptographic artifacts like certificates and private keys that are used for cryptographic protocols such as TLS. A truststore contains the certificate authority certificates which the endpoint trusts.
+
+The concept of the "truststore" is vital when it comes to secure communication with external services. It serves as a secure repository for storing certificates and trust anchors. In Zowe, the "truststore" is utilized to establish trust relationships with external services. The truststore captures and manages the relationship between Zowe's components and the certificates presented by the external services.
+
+In addition to utilizing the intra-address space of certificates, Zowe incorporates external services on z/OS to enhance the encryption of messages transmitted between its servers. These external services, such as z/OSMF or Zowe conformant extensions, have registered themselves with the API Mediation Layer.
+
+[Zowe API Mediation Layer](../user-guide/api-mediation/api-mediation-overview.md), acting as an intermediary, is responsible for validating these certificates. When the API Mediation Layer receives a certificate from an external service, it examines each certificate in the certificate chain and compares it to the certificates in the "truststore."
+
+By leveraging the truststore, Zowe ensures that only trusted and authorized external services can establish communication with its servers. The truststore validates the authenticity and integrity of the presented certificates, providing an additional layer of security.
+
+#### Keystore and Key ring
+
+Keystores are repositories that contain cryptographic artifacts like certificates and private keys that are used for cryptographic protocols such as TLS.
+
+A keystore contains personal certificates, plus the corresponding private keys that are used to identify the owner of the certificate.
+For TLS, a personal certificate represents the identity of a TLS endpoint. Both the client (for example, a REST client) and the server (for example, a IBM® z/OS® Connect server) might have personal certificates to identify themselves.
+
+What is key ring?
+
+#### Extended key usage
```

Comment:
to better follow our style guide: Extending key usage

Reply:
"Extended key usage" is a technical property name of a key, it's not about how we extend keys.